src/twisted/internet/test/test_tls.py

```diff
@@ -324,6 +325,26 @@ def useIt(reactor, contextFactory):
         self._testBadContext(useIt)
 
 
+    def connectToListener(self, reactor, address, factory):
+        """
+        Connect to the given listening TLS port, assuming the
+        underlying transport is TCP.
+
+        @param reactor: The reactor under test.
+        @type reactor: L{IReactorSSL}
+
+        @param address: The listening's address.
+        @type address: L{IPv4Address} or L{IPv6Address}
+
+        @param factory: The client factory.
+        @type factory: L{ClientFactory}
+
+        @return: The connector
+        """
+        return reactor.connectSSL(
+            '127.0.0.1', address.port, factory, self.getClientContext())
```

Comment:
[As above](https://github.com/twisted/twisted/pull/996/files/d29f39116cf69586ab814a366acca24f64fddc1a#r181253167).  These `connectToListener`s are special cases; the interface should in fact accept whatever address came out of `port.getHost()` so that the UNIX domain socket tests work without modification.  The easiest solution is probably a documentation change.

Reply:
Addressed in: https://github.com/twisted/twisted/pull/996/commits/963e05a2ef24c11b5d78cc385868f0d85602aaac